kokoro/linux/java_linkage_monitor/build.sh

```diff
@@ -6,7 +6,7 @@
 # This script selects a specific Dockerfile (for building a Docker image) and
 # a script to run inside that image.
 
-use_bazel.sh 5.1.1
+use_bazel.sh 4.0.0
```

Comment:
not 4.2.2?

Reply:
4.0.0 was the broken one, I'm not 100% certain 4.2.2 requires any fixes

Reply:
Note: it was only broken in our benchmark builds, so I have no way to confirm this fix actually works without downgrading a presubmit

Reply:
Upgraded us to 4.2.2 after offline chat